### PR TITLE
Feature/george/robolectric

### PR DESCRIPTION
**Pull request by zewolfe**

### CODE REVIEW

File: dhis2-android-app/src/test/java/org/dhis2/ehealthMobile/io/models/ModelUtils.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import android.os.Parcelable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by george on 1/4/17.
+ */
+
+public class ModelUtils {
```
Comment (lelloman) [inline dhis2-android-app/src/test/java/org/dhis2/ehealthMobile/io/models/ModelUtils.java]: setParcelId and setParcelLabel are identical, also the return value is never used, maybe just give up these 2 methods and use parce.writeString directly?

File: dhis2-android-app/src/test/java/IsTimelyTest.java [hunk 2]
```diff
@@ -9,6 +9,8 @@
 
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.core.Is.is;
```
Comment (lelloman) [inline dhis2-android-app/src/test/java/IsTimelyTest.java]: I like more human friendly semantic, check also this one https://github.com/square/assertj-android

Comment (zewolfe) [inline dhis2-android-app/src/test/java/IsTimelyTest.java]: +1 for this.

File: dhis2-android-app/src/test/java/org/dhis2/ehealthMobile/io/models/ModelUtils.java (hunk 1 above)
Comment (zewolfe) [inline dhis2-android-app/src/test/java/org/dhis2/ehealthMobile/io/models/ModelUtils.java]: Yeah. Makes sense.

**Pull request merged**